src/sentry/mediators/token_exchange/refresher.py

```diff
@@ -26,10 +27,24 @@ class Refresher(Mediator):
     user = Param(User)
     using = router.db_for_write(User)
 
+    @sentry_sdk.trace
```

Comment:
These operations should already occur with in a trace. Having a span for the Refresher.call() method seems reasonable to me though.

Reply:
My mistake :blush: